Error loading

Error on server start so plugin fails to load. (SlimeXpansion - DEV 2 (git 657f5))
```
[12:12:27] [Server thread/ERROR]: Could not load 'plugins\SlimeXpansion - DEV 2 (git 657f5).jar' in folder 'plugins'
org.bukkit.plugin.InvalidPluginException: Cannot find main class `me.john000708.SlimeXpansion'
	at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:65) ~[patched_1.14.4.jar:git-Paper-207]
	at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:134) ~[patched_1.14.4.jar:git-Paper-207]
	at org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.java:337) ~[patched_1.14.4.jar:git-Paper-207]
	at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:257) ~[patched_1.14.4.jar:git-Paper-207]
	at org.bukkit.craftbukkit.v1_14_R1.CraftServer.loadPlugins(CraftServer.java:355) ~[patched_1.14.4.jar:git-Paper-207]
	at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:226) ~[patched_1.14.4.jar:git-Paper-207]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:876) ~[patched_1.14.4.jar:git-Paper-207]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_221]
Caused by: java.lang.ClassNotFoundException: me.john000708.SlimeXpansion
	at java.net.URLClassLoader.findClass(Unknown Source) ~[?:1.8.0_221]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:139) ~[patched_1.14.4.jar:git-Paper-207]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:85) ~[patched_1.14.4.jar:git-Paper-207]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_221]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_221]
	at java.lang.Class.forName0(Native Method) ~[?:1.8.0_221]
	at java.lang.Class.forName(Unknown Source) ~[?:1.8.0_221]
	at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:63) ~[patched_1.14.4.jar:git-Paper-207]
	... 7 more

```
Paper v1_14_R1 (207)

CS-CoreLib vDEV - 77 (git 6e171836)
Slimefun vDEV - 274 (git 21582610)

Installed Addons (11)
 Barrels vDEV - 7 (git 0d14dd9f)
 ExtraGear vDEV - 3 (git de222ab7)
 ChestTerminal vDEV - 17 (git 933a7d36)
 EmeraldEnchants vDEV - 5 (git 6adbbb87)
 HotbarPets vDEV - 20 (git cd87bc60)
 ExoticGarden vDEV - 22 (git a855f5b7)
 SoulJars vDEV - 4 (git 4a600aa2)
 SlimefunOreChunks vDEV - 1 (git 87c8186b)
 ElectricSpawners vDEV - 5 (git c3f8cc9d)
 ExtraHeads v1.1
 ColoredEnderChests vDEV - 7 (git 6885cb88)

Still true for dev 3:
```
[10:58:02] [Server thread/ERROR]: Could not load 'plugins\SlimeXpansion - DEV 3 (git b57ba).jar' in folder 'plugins'
org.bukkit.plugin.InvalidPluginException: java.lang.NoClassDefFoundError: io/github/thebusybiscuit/cscorelib2/updater/Updater
	at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:138) ~[patched_1.14.4.jar:git-Paper-208]
	at org.bukkit.plugin.SimplePluginManager.loadPlugin(SimplePluginManager.java:337) ~[patched_1.14.4.jar:git-Paper-208]
	at org.bukkit.plugin.SimplePluginManager.loadPlugins(SimplePluginManager.java:257) ~[patched_1.14.4.jar:git-Paper-208]
	at org.bukkit.craftbukkit.v1_14_R1.CraftServer.loadPlugins(CraftServer.java:355) ~[patched_1.14.4.jar:git-Paper-208]
	at net.minecraft.server.v1_14_R1.DedicatedServer.init(DedicatedServer.java:226) ~[patched_1.14.4.jar:git-Paper-208]
	at net.minecraft.server.v1_14_R1.MinecraftServer.run(MinecraftServer.java:876) ~[patched_1.14.4.jar:git-Paper-208]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_221]
Caused by: java.lang.NoClassDefFoundError: io/github/thebusybiscuit/cscorelib2/updater/Updater
	at java.lang.Class.forName0(Native Method) ~[?:1.8.0_221]
	at java.lang.Class.forName(Unknown Source) ~[?:1.8.0_221]
	at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:63) ~[patched_1.14.4.jar:git-Paper-208]
	at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:134) ~[patched_1.14.4.jar:git-Paper-208]
	... 6 more
Caused by: java.lang.ClassNotFoundException: io.github.thebusybiscuit.cscorelib2.updater.Updater
	at java.net.URLClassLoader.findClass(Unknown Source) ~[?:1.8.0_221]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:139) ~[patched_1.14.4.jar:git-Paper-208]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:85) ~[patched_1.14.4.jar:git-Paper-208]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_221]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_221]
	at java.lang.Class.forName0(Native Method) ~[?:1.8.0_221]
	at java.lang.Class.forName(Unknown Source) ~[?:1.8.0_221]
	at org.bukkit.plugin.java.PluginClassLoader.<init>(PluginClassLoader.java:63) ~[patched_1.14.4.jar:git-Paper-208]
	at org.bukkit.plugin.java.JavaPluginLoader.loadPlugin(JavaPluginLoader.java:134) ~[patched_1.14.4.jar:git-Paper-208]
	... 6 more
```

--- a/src/me/john000708/slimexpansion/SlimeXpansion.java
+++ b/src/me/john000708/slimexpansion/SlimeXpansion.java
@@ -12,9 +12,6 @@ import org.bukkit.entity.Player;
 import org.bukkit.inventory.ItemStack;
 import org.bukkit.plugin.java.JavaPlugin;
 
-import io.github.thebusybiscuit.cscorelib2.updater.BukkitUpdater;
-import io.github.thebusybiscuit.cscorelib2.updater.GitHubBuildsUpdater;
-import io.github.thebusybiscuit.cscorelib2.updater.Updater;
 import me.john000708.slimexpansion.items.Linker;
 import me.john000708.slimexpansion.items.ScrapBox;
 import me.john000708.slimexpansion.listeners.ListenerSetup;
@@ -49,6 +46,9 @@ import me.mrCookieSlime.Slimefun.api.BlockStorage;
 import me.mrCookieSlime.Slimefun.api.Slimefun;
 import me.mrCookieSlime.Slimefun.bstats.bukkit.Metrics;
 import me.mrCookieSlime.Slimefun.cscorelib2.config.Config;
+import me.mrCookieSlime.Slimefun.cscorelib2.updater.BukkitUpdater;
+import me.mrCookieSlime.Slimefun.cscorelib2.updater.GitHubBuildsUpdater;
+import me.mrCookieSlime.Slimefun.cscorelib2.updater.Updater;
 
 /**
  * Created by John on 14.04.2016.

